src/components/snackbar/examples/snackbar-persistent.tsx

```diff
@@ -31,7 +31,7 @@ export class SnackbarPersistentExample {
                 message="We use cookies to deliver services and analyze traffic."
                 actionText="Okay, I got it!"
                 open={this.isOpen}
-                timeout={null}
+                timeout={-1}
```

Comment:
The comment above doesn't match this value, it mentions `null` as before this change, but nothing about `-1`.

Reply:
Does `null` and `-1` yield the same result?

Reply:
Yes, both work. I didn't see the comment mentioning `null` 😊

Reply:
c4f0b4e41168c884ed1f92382bb83fb863ae0557

Reply:
@jgroth had some opinions regarding `null` vs `-1´. I don't remember

Reply:
I did? :sweat_smile: I don't remember us talking about `-1` :sweat_smile:

Reply:
yes you did 😄 I don't exactly remember when, but perhaps it was when we were working on this issue https://github.com/Lundalogik/lime-elements/pull/2403/ or we were working on something about Snackbars; I remember that you thought it was more correct to have the timeout on `null`, instead of MDC's default value for getting a persistent timeout which was `-1`. Since it wasn't something that I personally had any opinions about, I forgot the discussion or your reasons… But I remember we were talking about it 😅